build.gradle

```diff
@@ -148,6 +148,8 @@ ext {
   springCloudVersion = '2021.0.3'
   tomcatVersion = '9.0.68'
   cxfVersion = '3.5.4'
+  activeMQVersion = '5.16.1'
+  testcontainers = '1.15.2'
```

Comment:
Before this variable was added the testcontainers were using version 1.17.5.  Should this be set to 1.17.5?

Reply:
changed to 1.17.5